### PR TITLE
Olwa pr

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Flexy OSIsoft Changelog
 
+## V2.7
+### Major Changes
++ None
+### Minor Changes
++ Fixed OMF bug in tag initialization
++ Made https timeout optional configuration parameter
++ OMF request responses are now saved serperatley by request type
```
Comment (TomKimsey) [inline docs/CHANGELOG.md:9]:
```suggestion
+ OMF request responses are now saved separately by request type
```

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Flexy OSIsoft Changelog
 
+## V2.7
+### Major Changes
++ None
+### Minor Changes
++ Fixed OMF bug in tag initialization
++ Made https timeout optional configuration parameter
```
Comment (TomKimsey) [inline docs/CHANGELOG.md:8]:
```suggestion
+ Made HTTPS timeout value an optional configuration parameter
```

**Review by TomKimsey — CHANGES_REQUESTED** [review]
I think the bug fix descriptions could use more detail. Its very vague what an "OMF bug in tag initialization" and "malformed post request" are.

**Comment by TomKimsey** [comment]
https://www.gnu.org/software/emacs/manual/html_node/emacs/Spelling.html

**Review by TomKimsey — APPROVED** [review]
not worth changing now but it might be good to add more information in the commit title for future commits

Updated changelog for v2.7
Rev'd version number to v2.7

That way in the repo there aren't 30 commits called "updated changelog".

**Comment by oliver-walker-hms** [comment]
that's a good point for the next one. I definitely need to update my emacs config file a little bit as well to cut down on typos. I want to get neo-tree back and set up a spell checker for most emacs modes Maybe during my upcoming vacation time i'll get that set up

**Pull request merged**